plan/IA.md

```diff
@@ -0,0 +1,162 @@
+# webcomponents.org
+
+## Goals
+
+- **Evangelize WCs**
+  - present the viability/maturity of Web Components
+    - general evangelism - #UseThePlatform, "works with all frameworks and none",
+    - "ways to make web components" : (lit, stencil, svelte, vanilla, ...registeredFeaturedTools)
+    - browser support
+    - popular framework features that can be implemented with Web Components
+      - SSR, SSG, etc explain only that there are ways to do it, not the "how, with code samples"
+- **Educate**
+  - Provide basic, generic MDN-like documentation for WC topics
+    - The WC specs explained - in our own words, not just linked to :)
+    - Creating a custom element
+      - "Writing web components" section from [Outline](/outline.md)
+    - Events
+    - Polyfills
+    - Scoped Elements
+    - Bundlers/bundling/#ToBundleOrNotToBundle
+      - Side Effects
+- **Showcase WCs**
+  - List "featured" components
+  - Component catalog/search
+- **Content Feed**
+  - Community articles about web components
+  - "anything goes" as long as its WC related
+  - can be tech stack, lib, approach -specific content
+- **Grow the community**
+  - join the WCCG
+  - submitting a WC to be "featured"
+  - writing a blogroll article
+  - writing an education page/article
+  - PRs, Issues to WCCG member projects
+- **Highlight WCCG recent work**
+  - Showcase work on adding to browser specs/features, RFCs etc
+  - Highlight Open WC, Modern Web, Rocket, etc written/maintained by folks in the WCCG
+- **About WCCG**
+  - Describe the WCCG and its mission, goals
+  - list team members
+
+
+## Audience
+
+- Users of comps
+- Devs of comps
+- Technology decider/architect/tech lead
+- Web component Novice devs
+- Experienced component devs
+
+
+## IA
+> ALL routes in this IA are definitely just examples for the ideas i'm presenting. Please don't treat them as finalized, but instead only as placeholders for the conceptual ideas about the content on a page with the name listed - MW
+
+### Simple IA
+Full with page descriptions [below](#full-ia)
+
+- Root /
+  - /our-work
```

Comment:
i didnt really touch on what would land in the nav vs page that would be accessible by other means. We could definitely divide up the links into "main nav" and "utility nav" though so that we only have a few items in the main nav.

imo stuff like "submit a component" and "become an author" totally dont belong in the main nav. I was just doing a sitemap style organization. i definitely didnt touch on navigation items, but thats something i can totally add!

Reply:
Ah, then indeed I may be a little ahead of the curve.  😅